2/scss/media-tablet.scss

```diff
@@ -0,0 +1,217 @@
+@import "vars";
+@import "mixins";
+
+@media (min-width: 768px) {
+  .hamburger-container {
+    margin: 28px 54px;
+  }
+
+  .header {
+    .container {
+      position: relative;
+    }
+  }
+
+  .logo {
+    margin: 28px 54px 0px 54px;
+    font-size: 24px;
+  }
+
+  .container-header-menu {
+    top: 75px;
+    z-index: 2;
+    .header-menu {
+      .header-menu-item{
+        padding: 13px;
+        margin-bottom: 0;
+      }
+    }
+  }
+
+  .header-text {
+    max-width: 1100px;
+    margin: 80px auto 0 auto;
+    padding: 20px;
+    .header-text-value{
+      font-size: 70px;
+      line-height: 84px;
+    }
+  }
+
+  .bottom-text-container {
+    flex-direction: row;
+    margin-top: 60px;
+    padding: 20px 20px 50px 20px;
+    .bottom-text {
+      @include text-size-small;
+      margin: auto;
+      padding-left: 30px;
+      border-left: 2px solid $colorTextPrimary;
+    }
+    .bottom-button-why-skout {
+      margin: auto;
+      .btn{
+        margin-left: 25px;
+        min-width: 200px;
+        min-height: 40px;
+      }
+    }
+  }
+
+  .about-skout-container {
+    margin: auto;
+    .skout-sa {
+      justify-content: center;
+      display: flex;
+      .skout-sa-img{
+        width: auto;
+      }
+    }
+    .about-skout {
+      .title-about-skout {
+        margin: 60px 0 30px 0;
+        @include text-size-medium;
+      }
+      .text-about-skout {
+        .text-about-skout-value{
```

Comment:
This will be compiled to `.text-about-skout .text-about-skout-value`

cause you have `.text-about-skout-value` only inside `.text-about-skout` long selector redundant here

&-value in this case would be better. Also this should be fixed all over the project.

Reply:
done